activerecord/lib/active_record/secure_token.rb

```diff
@@ -20,14 +20,35 @@ module ClassMethods
       #
       # <tt>SecureRandom::base58</tt> is used to generate the 24-character unique token, so collisions are highly unlikely.
       #
+      # A secure token can also be only created given a condition, for example if a user should only have an
+      # auto-generated invitation token if the user was invited:
+      #
+      #   # Schema: User(token:string, invited:boolean)
+      #   class User < ActiveRecord::Base
```

Comment:
@shunsukeaida It might be missed as `ApplicationRecord` is new thing in Rails 5 :smile: Please send PR